src/auxiliary/special_elixirs.jl

```diff
@@ -46,15 +46,14 @@ function convergence_test(mod::Module, elixir::AbstractString, iterations; kwarg
   # Types of errors to be calcuated
```

Comment:
(this comment is attached to the wrong line)

Can you please update the docstring such that it works not just for hierarchical meshes but also for curved meshes? I.e., from "increasing the refinement level" to "doubling the resolution" or something similar. And maybe instead of (initial) "refinement", make the names a little more general by talking about "resolution"? It's still not perfect, but I just would like the code to make sense to someone who comes upon ite without ever having used the tree but only the structured mesh.

`include_refined` is not perfect either, but I think it can stay as it is. Feel free to join it if you find a better name though.